index.bs

```diff
@@ -642,45 +681,6 @@ The {{VRDisplay}} associated with this event.
 <dfn attribute for="VRDisplayEvent">reason</dfn>
 {{VRDisplayEventReason}} describing why this event has has been fired.
 
-
-## Window Interface extension ## {#interface-window}
-
-<pre class="idl">
-partial interface Window {
-  attribute EventHandler onvrdisplayconnect;
-  attribute EventHandler onvrdisplaydisconnect;
-  attribute EventHandler onvrdisplayactivate;
```

Comment:
Perhaps this is a silly question... but where did this portion of the idl get moved to?  I thought the events were going to be added to VRDisplay and while I see the comments moved above, but I don't see the EventHandlers themselves.  Am I misunderstanding how this works?